test-failure-policy

```diff
@@ -66,6 +68,12 @@ def main():
             print("due to failure of test harness or framework")
             return 1
 
+        if not number and opts.all:
+            number, img = check_all_known_issues(api, output, opts.image)
+            if number:
+                print(f"Known issue #{number} in {img}")
+                return 78
```

Comment:
This isn't a magic number, but I don't mind much.

Reply:
Nevermind, it becomes magic in https://github.com/cockpit-project/cockpit/pull/19173